docs/components/board/upboard.md

```diff
@@ -10,6 +10,7 @@ tags: ["board", "components"]
 ---
 
 Configure an `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot.
+Complete the following set-up requirements, then move on to configuring your board in [the Viam app](https://app.viam.com):
```

Comment:
```suggestion

Complete the following setup requirements, then move on to configuring your board in [the Viam app](https://app.viam.com):
```
I think we've been doing setup as one word when it's a noun like this. LGTM thx for adding.